CHANGES/5278.misc

```diff
@@ -0,0 +1 @@
+Drop Python 3.6 support -- by :user:`asvetlov`
```

Comment:
@bdraco there's 2 drop notes in the 3.9.0: https://docs.aiohttp.org/en/latest/changes.html#id24. Perhaps, this shouldn't be the third one?

Reply:
removed in e0b3542b2a4ecd027a0e45e422a1df48d5c93477